### PR TITLE
Document JSON rendering behavior for durations

### PR DESCRIPTION
**Pull request by dominiklohmann**
This just documents existing behavior.

### :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Read the new help text.

### CODE REVIEW

File: docs/cli/vast-export-json.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 The JSON export format renders events in newline-delimited JSON (aka.
 [JSONL](https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON)).
+
+By default, durations render as human-readable strings with up to two decimal
+places and a unit, e.g., `2.31s` or `24.1ms`. The option `--numeric-durations`
+causes VAST to render durations as numbers instead, representing the time in
+nanoseconds since the UNIX epoch.
```
Comment (tobim) [inline docs/cli/vast-export-json.md:7]: This sounds wrong to me. If the value is already a duration we have a start point already, replacing that with epoch doesn't make sense.

Comment (dominiklohmann) [reply docs/cli/vast-export-json.md:7]: Eh, brain fart moment, that's how we'd render timestamps. You saw nothing!

File: docs/cli/vast-export-json.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
 The JSON export format renders events in newline-delimited JSON (aka.
 [JSONL](https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON)).
+
+By default, durations render as human-readable strings with up to two decimal
+places and a unit, e.g., `2.31s` or `24.1ms`. The option `--numeric-durations`
```
Comment (mavam) [inline docs/cli/vast-export-json.md:5]:
```suggestion
By default, durations render as human-readable strings rounded up to two decimal
digits and a unit, e.g., `2.31s` or `24.1ms`. The option `--numeric-durations`
```

File: libvast/src/system/application.cpp [hunk 3]
```diff
@@ -128,7 +128,8 @@ auto make_export_command() {
       .add<bool>("flatten", "flatten nested objects into "
                             "the top-level")
       .add<bool>("numeric-durations", "render durations as numbers as opposed "
-                                      "to human-readable strings")
+                                      "to human-readable strings with up to "
+                                      "two decimal places")
```
Comment (mavam) [inline libvast/src/system/application.cpp:132]:
```suggestion
                                      "to human-readable strings rounded up to "
                                      "two decimal digits")
```

**Review by mavam — APPROVED** [review]

File: docs/cli/vast-export-json.md (hunk 2 above)
Comment (tobim) [reply docs/cli/vast-export-json.md:5]: "rounded up" can be taken quite literal.

Comment (mavam) [reply docs/cli/vast-export-json.md:5]: What is the issue with this?

Aside from it, we should say simply "rounded to..." without "up".

Comment (dominiklohmann) [reply docs/cli/vast-export-json.md:5]: Isn't "decimal places" correct over "decimal digits"? I've never heard the latter.

Comment (mavam) [reply docs/cli/vast-export-json.md:5]: I think "place" is indeed more common. Personally, I find "digit" more visual and easier to grasp, but then this doesn't matter if "decimal place" is the common expectation.

Consider my feedback on this as discarded.

**Pull request merged**